BE/db/seed.js

```diff
@@ -1,3 +1,5 @@
+const bcrypt = require("bcrypt");
```

Comment:
Why is this stuff I added showing up in your PR?

Reply:
haha i added it. I needed it for testing. You can't stop me boiiiiiiiiiii